src/components/Dialogs/PeriodDimension/StartEndDate.js

```diff
@@ -10,7 +10,7 @@ export const StartEndDate = ({ value, setValue }) => {
     const [endDate, setEndDate] = useState(endDateStr)
 
     useEffect(() => {
-        setValue(startDate && endDate ? `${startDate}_${endDate}` : '')
+        setValue(startDate && endDate ? `${startDate}_${endDate}` : null)
```

Comment:
If you want, you can stick with empty string for both here and line 152 of PeriodDImension, or null in both places?